
# Timestap of the microcontroller, in millisecs since boot
uint32  timestamp_ms
# Vehicle speed
float64   speed_vehicle_ecarm